VTUL/compel
# Update gems to address security issues

**Update gems to address security issues.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1869) (:star:)

# What does this Pull Request do? (:star:)
Update gems to address security issues.

# What's the changes? (:star:)
* Update gems to address security issues.

# How should this be tested?
* set `ruby_version: '2.4'` in Installscripts/ansible/site_secrets.yml
* Build `gem_updates` branch and ensure that application is still functional

# Additional Notes:
* branch gem_updates

# Interested parties
@pmather 

(:star:) Required fields
